### ISSUE TITLE
Lexical error with tab character in literate Agda text

### ISSUE THREAD

**Issue by wadler** [agda/agda#5581]
Agda reports a lexical error when there is a tab in the source code, even if the tab appears in a commented-out part of the source.

**Comment by andreasabel** [comment]
Phil @wadler, would you attach a file with MWE?  I cannot reproduce this, tabs in comments seem ok.

**Comment by nad** [comment]
@andreasabel, try putting the following text into `Bug.lagda.md`:
```
A tab character: '	'.
```
Result:
```
Bug.lagda.md:1,1: Lexical error (you may want to replace tabs with spaces):
<ERROR>
	  
...
```

**Comment by andreasabel** [comment]
The problem is that literate stuff in a `.lagda` file is _bleached_, meaning non-whitespace is replaced by space characters.  However, tabs, being ws, are kept, failing the Agda lexer.
https://github.com/agda/agda/blob/78ea6d259b10d4751f96924e418dbc1c8cd8b638/src/full/Agda/Syntax/Parser/Literate.hs#L124-L136
A fix would be to replace tabs by single spaces, that would preserve the character position.

**Comment by andreasabel** [comment]
> A fix would be to replace tabs by single spaces, that would preserve the character position.

Implemented this in #5582.
